test/beginend-prog-test.el

```diff
@@ -82,14 +82,22 @@
         (expect (beginend--prog-mode-code-position-p) :to-be-truthy)))
 
     (describe "returns nil"
-      (it "for a line of comment"
+      (it "for a comment line"
```

Comment:
I took the liberty of changing the wording here. I hope that is okay.

Reply:
it is, thank you